wasm_exec.js

```diff
@@ -1,8 +1,15 @@
+// NOTE: This package is using esbuild with a plugin that inlines WASM.
+//    Because of this the "fs" module is no longer needed, and the polyfill
+//    for that has been removed, along with any need to detect the existence
+//    of a global `require`.
+
 // Copyright 2018 The Go Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 //
 // This file has been modified for use by the TinyGo compiler.
+//
+// This file has been modified for use by Tableland.
```

Comment:
I wasn't able to find a way to get esbuild to work for all builds, and environments, without modifying this file.
Also as noted in the comment above, this file used to containe a polyfill for the `fs` package. `fs` was only used (afaict) to load the wasm file. The wasm file is now being inlined by our esbuild plugin, so the need for `fs` should be gone.